client/state/data-layer/wpcom/read/streams/index.js

```diff
@@ -193,6 +193,21 @@ const streamApis = {
 	following: {
 		path: () => '/read/following',
 		dateProperty: 'date',
+		query: ( extras ) => {
+			// Filter out undefined values from extras
+			const filteredExtras = Object.fromEntries(
+				Object.entries( extras ).filter( ( [ , value ] ) => value !== undefined )
+			);
+
+			// If no after param is provided, default to 60 days ago
+			if ( ! filteredExtras.after && ! filteredExtras.pageHandle?.after ) {
+				const sixtyDaysAgo = new Date();
+				sixtyDaysAgo.setDate( sixtyDaysAgo.getDate() - 60 );
+				filteredExtras.after = sixtyDaysAgo.toISOString().split( '.' )[ 0 ] + 'Z';
+			}
```

Comment:
Here, we set the `after` parameter to get posts less than 60 days old. In theory, you can set the `after` param in the pageHandle, as seen in [this existing unit test](https://github.com/Automattic/wp-calypso/pull/98944/files#diff-ab93180c1ff1b63f08d736045f7639b1d56933e42d505741ee6439728221490bR48). For this reason, we only apply the 60-day default if the `after` parameter isn't already set.